addons/website_event/static/tests/tours/website_event.js

```diff
@@ -26,16 +24,12 @@ odoo.define("website_event.tour", function (require) {
         run: 'text Technical Training',
         position: "left",
     }, {
-        content: _t("Open date range picker."),
-        trigger: '.modal-dialog [name=date_begin] input',
-    }, {
-        trigger: '.modal-dialog [name=date_begin] input',
-        extra_trigger: '.daterangepicker',
-        content: _t("Pick a Start date for your event"),
+        trigger: '.modal-dialog div[name=date_begin]',
+        content: _t("Open date range picker. Pick a Start date for your event"),
```

Comment:
Doesn't that also generate a string in website_event pot file ?

Reply:
It does not seem to generate a string in website_event pot file . I think it's because it's a test tour, there seem to be no translations for other test tour strings either